src/index.js

```diff
@@ -1,7 +1,55 @@
 // Importing CSS styles
 import './style.css';
-import render from './modules/displaylist.js';
+import {
+  render,
+  addTask,
+  editTask,
+  removeTask,
+} from './modules/handletodo.js';
 
-// Adding main component to the HTML file
-const mainContainer = document.querySelector('.main');
-mainContainer.appendChild(render());
+render();
+// add task when text is submitted
+const taskinput = document.querySelector('.task-desc');
```

Comment:
[Optional] Currently, your app refreshes every time a task is added and removed from the list. To improve the user experience, I suggest you prevent this refresh action.